src/components/atoms/OcButton/OcButton.vue

```diff
@@ -56,6 +56,13 @@ export default {
       type: String,
       default: null,
     },
+    /**
+     * When setting the button’s type to a link, use this option to give a give a target.
+     */
+    target: {
+      type: String,
+      default: null,
```

Comment:
Super nitpicky, but shouldn't we only allow the few options available in the HTML standard? https://www.w3schools.com/tags/att_a_target.asp

Reply:
theoretically you could use an iframe as an target, but i think we should validate because thats pretty rare.